test/integ_tests/test_all_notebooks.py

```diff
@@ -0,0 +1,100 @@
+import logging
+import os
+import pytest
+
+from testbook import testbook
+from importlib.machinery import SourceFileLoader
+
+# These notebooks have syntax or dependency issues that prevent them from being tested.
+EXCLUDED_NOTEBOOKS = [
+    # These notebooks have cells that have syntax errors
+    "4_Operating_Borealis_beginner_tutorial.ipynb",
+    "bring_your_own_container.ipynb",
+    "qnspsa_with_embedded_simulator.ipynb",
+    # These notebooks have dependency issues
+    "VQE_chemistry_braket.ipynb",
+    "6_Adjoint_gradient_computation.ipynb",
```

Comment:
Are we planning on fixing these in a follow up PR ? If yes, would we then be able to use the same requirements file across the repository ?